go.mod

```diff
@@ -67,7 +67,7 @@ require (
 	github.com/prometheus/client_golang v1.13.0
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.37.0
-	github.com/prometheus/exporter-toolkit v0.7.1
+	github.com/prometheus/exporter-toolkit v0.8.2
```

Comment:
I think this breaks exporter-toolkit API a bit. 

It was bumped to `v0.8.1` [here](https://github.com/thanos-io/thanos/pull/5896/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6L70) and the fix for that is [here](https://github.com/thanos-io/thanos/pull/5896/files#diff-50f2de48612be70aae1c45c5f804554a71b4e7b846ec3f137e68cfe07a04785dR79). Maybe that can be done on this PR instead? 🙂